src/mawpy/steps/incremental_clustering.py

```diff
@@ -26,17 +26,22 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_cluster_center(row: pd.Series, mapping: dict, dur_constr: float) -> tuple[str, str, str]:
+def _get_cluster_center(row: pd.Series, df_columns: list, mapping: dict, dur_constr: float) -> tuple[str, str, str]:
     """
     Returns the lat and long of the cluster to which the trace(row) belongs.
     """
     if dur_constr:
         lat_long = (row[STAY_LAT], row[STAY_LONG])
-        unc = row[STAY_UNC]
```

Comment:
Could this be simplified by adding `STAY_UNC` column with default value `-1` when the file is initially loaded if the column is not present? I'm not sure if such columns are stored in a memory efficient way by pandas but probably worth trying.

Reply:
Yes, was thinking the same. Will have to crosscheck for accuracy and timing as well. 

Meanwhile, created an [issue](https://github.com/uw-ssec/MAWpy/issues/68) to explore this separately.